<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWin</class>
 <widget class="QMainWindow" name="LoginWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>818</width>
    <height>282</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>LoginWin</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{
	background-color: rgb(255, 254, 240);
	border-image: url(:/new/prefix1/image/back1.jpg);
}
 
QLineEdit{
 
	border:2px solid#7b7b7b;
	border-radius:15px;
	background-color:#fffef0
}

QComboBox{
 
	border:2px solid#7b7b7b;
	border-radius:15px;
	background-color:#fffef0
}

QComboBox{
 
	border:2px solid#7b7b7b;
	border-radius:15px;
	background-color:#fffef0
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>1</number>
      </property>
      <property name="topMargin">
       <number>1</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <property name="horizontalSpacing">
       <number>6</number>
      </property>
      <item row="2" column="3" colspan="2">
       <widget class="QLabel" name="label_fxgc">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>Agency FB</family>
          <pointsize>18</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>分项工程：</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QLineEdit" name="lineEdit_xmdm">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>Arial Black</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="5">
       <widget class="QLabel" name="label_title">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>黑体</family>
          <pointsize>28</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(9, 41, 4, 255), stop:0.085 rgba(2, 79, 0, 255), stop:0.19 rgba(50, 147, 22, 255), stop:0.275 rgba(236, 191, 49, 255), stop:0.39 rgba(243, 61, 34, 255), stop:0.555 rgba(135, 81, 60, 255), stop:0.667 rgba(121, 75, 255, 255), stop:0.825 rgba(164, 255, 244, 255), stop:0.885 rgba(104, 222, 71, 255), stop:1 rgba(93, 128, 0, 255));
font: 28pt &quot;黑体&quot;;
color:rgb(255, 255, 0)</string>
        </property>
        <property name="text">
         <string>  欢迎使用编码设计小工具</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_gclx">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>Agency FB</family>
          <pointsize>18</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>工程类型：</string>
        </property>
       </widget>
      </item>
      <item row="2" column="5">
       <widget class="QComboBox" name="comboBox_fxgc">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">
QComboBox {
	border: 1px solid #bebebe;
	padding: 1px 18px 1px 3px;
	font: normal normal 16px &quot;Microsoft YaHei&quot;;
	color: #555555;

}
 

 
QComboBox:!editable:on, QComboBox::drop-down:editable:on{
	background: transparent;
}
 
QComboBox:!on{
}
 
QComboBox:on{ /* the popup opens */
	color: #555555;
	border-color: #327cc0;
	background: transparent;
}
 
QComboBox::drop-down{
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 20px;
    border-left-width: 1px;
    border-left-color: darkgray;
}
 
QComboBox::down-arrow {
	image: url(:/common/combobox_arrow);
}
 
QComboBox::down-arrow:on {
	image: url(:/common/combobox_arrow_up);
}
 
QComboBox QAbstractItemView {
	outline: 0; 
	border: 1px solid #327cc0;
	background-color: #F1F3F3;
	font: normal normal 14px &quot;Microsoft YaHei&quot;;
}
 
QComboBox QAbstractItemView::item {
	height: 32px;
	color: #555555;
	background-color: transparent;
}
 
QComboBox QAbstractItemView::item:hover {
	color: #FFFFFF;
	background-color: #327cc0;
}
 
QComboBox QAbstractItemView::item:selected {
	color: #FFFFFF;
	background-color: #327cc0;
}
 
QComboBox QAbstractScrollArea QScrollBar:vertical {
	background-color: #d0d2d4;
}
 
QComboBox QAbstractScrollArea QScrollBar::handle:vertical {
	background: rgb(160,160,160);
}
 
QComboBox QAbstractScrollArea QScrollBar::handle:vertical:hover {
	background: rgb(90, 91, 93);
}</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3" colspan="2">
       <widget class="QLabel" name="label_jddm">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>Agency FB</family>
          <pointsize>18</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>阶段代码：</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="2">
       <widget class="QComboBox" name="comboBox_gclx">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>Microsoft YaHei</family>
          <pointsize>-1</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">
QComboBox {
	border: 1px solid #bebebe;
	padding: 1px 18px 1px 3px;
	font: normal normal 16px &quot;Microsoft YaHei&quot;;
	color: #555555;

}
 

 
QComboBox:!editable:on, QComboBox::drop-down:editable:on{
	background: transparent;
}
 
QComboBox:!on{
}
 
QComboBox:on{ /* the popup opens */
	color: #555555;
	border-color: #327cc0;
	background: transparent;
}
 
QComboBox::drop-down{
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 20px;
    border-left-width: 1px;
    border-left-color: darkgray;
}
 
QComboBox::down-arrow {
	image: url(:/common/combobox_arrow);
}
 
QComboBox::down-arrow:on {
	image: url(:/common/combobox_arrow_up);
}
 
QComboBox QAbstractItemView {
	outline: 0; 
	border: 1px solid #327cc0;
	background-color: #F1F3F3;
	font: normal normal 14px &quot;Microsoft YaHei&quot;;
}
 
QComboBox QAbstractItemView::item {
	height: 32px;
	color: #555555;
	background-color: transparent;
}
 
QComboBox QAbstractItemView::item:hover {
	color: #FFFFFF;
	background-color: #327cc0;
}
 
QComboBox QAbstractItemView::item:selected {
	color: #FFFFFF;
	background-color: #327cc0;
}
 
QComboBox QAbstractScrollArea QScrollBar:vertical {
	background-color: #d0d2d4;
}
 
QComboBox QAbstractScrollArea QScrollBar::handle:vertical {
	background: rgb(160,160,160);
}
 
QComboBox QAbstractScrollArea QScrollBar::handle:vertical:hover {
	background: rgb(90, 91, 93);
}</string>
        </property>
       </widget>
      </item>
      <item row="5" column="5">
       <widget class="QLabel" name="label_author">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>Agency FB</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>Author：LI WEIYUAN</string>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="QLineEdit" name="lineEdit_jddm">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>Arial Black</family>
          <pointsize>18</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item row="6" column="5">
       <widget class="QLabel" name="label_version">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>Agency FB</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>Version：1.3-2020.12.03</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_xmdm">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>Agency FB</family>
          <pointsize>18</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>项目代码：</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="4" column="1" colspan="2">
       <widget class="QPushButton" name="btn1">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>幼圆</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>读取配置文件</string>
        </property>
       </widget>
      </item>
      <item row="4" column="3" colspan="3">
       <widget class="QPushButton" name="btn2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>幼圆</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>开始生成</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1" colspan="2">
       <widget class="QLabel" name="label">
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 255, 0);</string>
        </property>
        <property name="text">
         <string>注意：请使用含有固定格式的配置文件</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
